zulipterminal/ui_tools/buttons.py

```diff
@@ -26,7 +26,7 @@ def update_count(self, count: int) -> None:
 
     def widget(self, count: int) -> Any:
         return urwid.AttrMap(urwid.SelectableIcon(
-            [u' 🏡 ', self.caption,
+            [u'\N{BULLET} ', self.caption,
```

Comment:
Just adding a space before the bullet seems to make it look better. 

![image](https://user-images.githubusercontent.com/315678/42492201-cdc2a6ca-8435-11e8-952e-5b30544db752.png)

![image](https://user-images.githubusercontent.com/315678/42492230-e95bd2b2-8435-11e8-9110-a03d8a641613.png)